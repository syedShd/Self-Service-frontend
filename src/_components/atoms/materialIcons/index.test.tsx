import { render } from '@testing-library/react'
import Icon from '.'
import { screen } from '@testing-library/react'

test('select color icon component should be displayed', () => {
  render(<Icon style={undefined} className={undefined} iconColor="select" type="filled" />)
  const icon = screen.getByTestId('icon')
  expect(icon).toBeInTheDocument()
})
test('unselect color icon component should be displayed ', () => {
  render(<Icon style={undefined} className={undefined} iconColor="unselect" type="outlined"/>)
  const icon = screen.getByTestId('icon')
  expect(icon).toBeInTheDocument()
})
test('info icon color component should be displayed', () => {
  render(<Icon style={undefined} className={undefined} iconColor="info" type="filled"/>)
  const icon = screen.getByTestId('icon')
  expect(icon).toBeInTheDocument()
})
test(' success icon color component should be displayed', () => {
  render(<Icon style={undefined} className={undefined} iconColor="success" type="filled"/>)
  const icon = screen.getByTestId('icon')
  expect(icon).toBeInTheDocument()
})
test('default icon color component should be displayed', () => {
  render(<Icon style={undefined} className={undefined} iconColor="test"type="filled"/>)
  const icon = screen.getByTestId('icon')
  expect(icon).toBeInTheDocument()
})




