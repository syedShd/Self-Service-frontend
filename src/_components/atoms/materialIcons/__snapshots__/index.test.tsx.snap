// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon testing Matches snapshot 1`] = `
<div>
  <span
    aria-hidden="true"
    class="material-icons MuiIcon-root  material-icons-outlined"
    style="color: rgb(0, 0, 0);"
  />
</div>
`;
